app/_hub/kong-inc/rate-limiting-advanced/_index.md

```diff
@@ -463,6 +463,14 @@ selected header was not sent by the client or the configured service was not fou
 
 [`Retry-After`]: https://tools.ietf.org/html/rfc7231#section-7.1.3
 
+### Fallback from Redis
+
+When the `redis` strategy is used and a Kong node gets disconnected from Redis (Redis server down,
+connection to Redis broken, etc), the `rate-limiting-advanced` plugin will fall back to `local`.
```

Comment:
```suggestion
When the `redis` strategy is used and a {{site.base_gateway}} node is disconnected from Redis, the `rate-limiting-advanced` plugin will fall back to `local`. This can happen when the Redis server is down or the connection to Redis broken.
```